milestone6.md

```diff
@@ -0,0 +1,23 @@
+# Milestone 6
+
+## Current UI Screenshots
+
+### Home Screenshots
+
+![Home Screen](/images/milestone6/home1.png)
+
+![Home Screen](/images/milestone6/home2.png)
+
+### Listings Screenshot
+
+![List Screen](/images/milestone6/listings-page.png)
```

Comment:
I'm not sure if we will still need 2 screenshots for this portion as the listings are all displayed on a single page.

Reply:
For this part the other screenshot is a gif so that we could show the interactivity of the map and first image is a static image just showing the page.